qiskit_optimization/runtime/qaoa_client.py

```diff
@@ -72,7 +78,29 @@ def __init__(
                 ansatz, the evaluated mean and the evaluated standard deviation.
             store_intermediate: Whether or not to store intermediate values of the optimization
                 steps. Per default False.
+            use_swap_strategies: A boolean on whether or not to use swap strategies when
+                transpiling. If this is False then the standard transpiler with the given
+                optimization level will run.
+            use_initial_mapping: A boolean flag that, if set to True (the default is False), runs
+                a heuristic algorithm to permute the Paulis in the cost operator to better fit the
+                coupling map and the swap strategy. This is only needed when the optimization
+                problem is sparse and when using swap strategies to transpile.
+            use_pulse_efficient: A boolean on whether or not to use a pulse-efficient transpilation.
+                If this flag is set to False by default. See https://arxiv.org/abs/2105.01063.
+            optimization_level: The transpiler optimization level to run if the swap strategies are
+                not used. This value defaults to 1 in the QAOA runtime.
+
+        Raises:
+            QiskitOptimizationError: if reps is smaller than 1.
+            QiskitOptimizationError: if optimization_level is not None and use_swap_strategies
+                is True.
         """
+        if reps < 1:
+            raise QiskitOptimizationError(f"reps must be greater than 0, received {reps}.")
+
+        if initial_point is None:
+            initial_point = np.random.rand(2 * reps)
```

Comment:
I would suggest the following based on this comment https://github.com/Qiskit/qiskit-optimization/pull/288#discussion_r769816567.
```suggestion
            initial_point = np.random.default_rng().random(2 * reps)
```

Reply:
This was remove entirely. See here: 2f5a09b